### PR TITLE
CHT Gateway Doc Update

### PR DESCRIPTION
**Pull request by billwambua**
Make CHT Gateway doc easier to follow and correct spelling mistakes.

### CODE REVIEW

File: content/en/apps/guides/messaging/gateways/gateway/configuration.md [hunk 1]
```diff
@@ -11,26 +11,26 @@ relatedContent: >
...
 
-- Open the app.
-You will get a ```Warning:medic-gateway is not set as the default messaging app on this device``` if you are installing the app for the first time or afresh. Select ```HELP ME CHANGE``` and agree to the folloup system prompt about changing the default messaging app.
+2. Open the app.
+if you are installing the app for the first time or afresh, you will get a ```Warning:medic-gateway is not set as the default messaging app on this device``` . Select ```HELP ME CHANGE``` and agree to the follow-up system prompt about changing the default messaging app.
 
-- If you're configuring medic-gateway (v1.0.0 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```my_gateway_user``` and a password of ```topSecret```, fill in the settings as follows:
+3. Configure the app. If you're configuring cht-gateway(v1.00 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```gateway``` and a password of ```topSecret```, fill in the settings as follows:
```
Comment (garethbowen) [inline content/en/apps/guides/messaging/gateways/gateway/configuration.md:21]: I think the version should be `1.0.0` like it was before. `1.00` looks weird to me!

```suggestion
3. Configure the app. If you're configuring cht-gateway(v1.0.0 and above) for use with hosted medic, with a URL of e.g. ```https://myproject.dev.medicmobile.org``` and a username of ```gateway``` and a password of ```topSecret```, fill in the settings as follows:
```

File: content/en/apps/guides/messaging/gateways/gateway/configuration.md [hunk 2]
```diff
@@ -11,26 +11,26 @@ relatedContent: >
...
 
 
 Instance name: `myproject [dev]`   (if ```https://myproject.app...```, select 'app')<br>
-Username: `my_gateway_user`<br>
-Password: `topSecret`
+Username: `gateway`(_Since_ [V1.2.1](https://github.com/medic/cht-gateway/releases/tag/v1.2.1) _medic gateway versions this field is not present in the app. The user `gateway` is assumed_.)<br>
```
Comment (garethbowen) [inline content/en/apps/guides/messaging/gateways/gateway/configuration.md:25]: Minor suggestion...

```suggestion
Username: `gateway`(_Since_ [v1.2.1](https://github.com/medic/cht-gateway/releases/tag/v1.2.1) _medic gateway versions this field is not present in the app. The user `gateway` is assumed_.)<br>
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Thanks! A couple of small suggestions inline.

Also, it looks like you've added an image but haven't included it in the page anywhere?

**Comment by billwambua** [comment]
> Thanks! A couple of small suggestions inline.
> 
> Also, it looks like you've added an image but haven't included it in the page anywhere?

About the image, what I did was replace the existing image(`gateway-config.png`) with a screenshot of the newest CHT-Gateway version.

**Review by garethbowen — APPROVED** [review]
Awesome, thanks!

**Comment by kennsippell** [comment]
Nice one @billwambua!

**Pull request merged**